.gitignore

```diff
@@ -1,3 +1,4 @@
 .*.cache
+.idea/
```

Comment:
this is related to your IDE setup, so this should be ignored thanks to file specific to your computer (either only for that project in `.git/info/exclude` or through a system-wide ignore rule). See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files for the various ways to exclude files in git.

We don't want to maintain exclude rules for all IDEs out there in the project.

Reply:
updated, thanks for the advices :)